ir/instr.cpp

```diff
@@ -8,6 +8,7 @@
 #include "smt/solver.h"
 #include "util/compiler.h"
 #include <functional>
+#include <iostream>
```

Comment:
this is probably a debug leftover

Reply:
Thank you for your checking.
I fixed it and removed unused argument in the 'Calloc' class and a test.